sympy/physics/mechanics/functions.py

```diff
@@ -299,44 +299,16 @@ def potential_energy(*body):
 
 
 def gravity(acceleration, *bodies):
-    """
-    Returns a list of gravity forces given the acceleration
-    due to gravity and any number of particles or rigidbodies.
-
-    Example
-    =======
-
-    >>> from sympy.physics.mechanics import ReferenceFrame, Point, Particle, outer, RigidBody
-    >>> from sympy.physics.mechanics.functions import gravity
-    >>> from sympy import symbols
-    >>> N = ReferenceFrame('N')
-    >>> m, M, g = symbols('m M g')
-    >>> F1, F2 = symbols('F1 F2')
-    >>> po = Point('po')
-    >>> pa = Particle('pa', po, m)
-    >>> A = ReferenceFrame('A')
-    >>> P = Point('P')
-    >>> I = outer(A.x, A.x)
-    >>> B = RigidBody('B', P, A, M, (I, P))
-    >>> forceList = [(po, F1), (P, F2)]
-    >>> forceList.extend(gravity(g*N.y, pa, B))
-    >>> forceList
-    [(po, F1), (P, F2), (po, g*m*N.y), (P, M*g*N.y)]
-
-    """
-
-    gravity_force = []
-    if not bodies:
-        raise TypeError("No bodies(instances of Particle or Rigidbody) were passed.")
-
-    for e in bodies:
-        point = getattr(e, 'masscenter', None)
-        if point is None:
-            point = e.point
-
-        gravity_force.append((point, e.mass*acceleration))
-
-    return gravity_force
+    from sympy.physics.mechanics.loads import gravity as _gravity
+    sympy_deprecation_warning(
+        """
+        The gravity function has been moved.
+        Import it from "sympy.physics.mechanics.loads".
```

Comment:
Do we want to suggest people import it from `sympy.physics.mechanics.loads` or do we want to promote it as part of the formal API and suggest it's imported from `sympy.physics.mechanics`?

Reply:
Depends whether we really want to promote this function. Personally, I don't see the need for promoting it, because it is more sensible to have an `apply_gravity` method in the future `System` class, which uses this function. I'm not sure what is the reason it was never promoted, so I'd rather keep it that way.